### PR TITLE
user/moar: new package

### PR DESCRIPTION
**Pull request by Biswa96**

### CODE REVIEW

File: user/moar/template.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+maintainer = "Biswapriyo Nath <nathbappai@gmail.com>"
+license = "BSD-2-Clause"
+url = "https://github.com/walles/moar"
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "a26954fca06c95604a94b6f208f62eff247e50c8d239262322a13d0959074de1"
+
+def install(self):
+    self.install_bin("build/moar")
+    self.install_license("LICENSE")
+    self.install_man("moar.1", glob=True)
```
Comment (q66) [inline user/moar/template.py:19]: the glob here does not seem to serve any purpose?

File: user/moar/template.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    f"-ldflags=-X main.versionString=v{pkgver}",
+]
+hostmakedepends = ["go"]
+pkgdesc = "Pager designed to just do the right thing without any configuration"
+maintainer = "Biswapriyo Nath <nathbappai@gmail.com>"
+license = "BSD-2-Clause"
+url = "https://github.com/walles/moar"
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "a26954fca06c95604a94b6f208f62eff247e50c8d239262322a13d0959074de1"
+
```
Comment (q66) [inline user/moar/template.py:15]: format your template with black or ruff

File: user/moar/template.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+pkgname = "moar"
+pkgver = "1.30.0"
+pkgrel = 0
+build_style = "go"
+make_build_args = [
```
Comment (q66) [inline user/moar/template.py:5]: it should be ok for this to go on one line

File: user/moar/template.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    f"-ldflags=-X main.versionString=v{pkgver}",
+]
+hostmakedepends = ["go"]
+pkgdesc = "Pager designed to just do the right thing without any configuration"
```
Comment (q66) [inline user/moar/template.py:9]: i'd prefer a terser and less opinionated description (most people don't configure `less` either), so just `Terminal pager program` should be fine

**Comment by Biswa96** [comment]
All requested changes have been made, please review the pull request.

**Comment by q66** [comment]
lgtm

**Pull request merged**